convey/stack_trace_test.go

```diff
@@ -119,13 +129,13 @@ type memoryFile struct {
 	buffer string
 }
 
-func (self *memoryFile) Write(p []byte) (n int, err error) {
-	self.buffer += string(p)
+func (mf *memoryFile) Write(p []byte) (n int, err error) {
```

Comment:
👍

Reply:
yeah, I saw a lot of "this" and "self" usage, which my IDE and my eyes don't really like..I'll update more in a style-only PR